src/daft-plan/src/logical_ops/source.rs

```diff
@@ -53,8 +53,7 @@ impl Source {
                 partitioning_keys,
                 pushdowns,
             })) => {
-                res.push("Source:".to_string());
-                res.push(format!("Scan op = {}", scan_op));
+                res.push(format!("Source: Operator = {}", scan_op));
```

Comment:
Is this more readable when visualizing the plan? In the past, we've tried to keep the first line of the logical op repr super concise, essentially just a name for the logical op, but it looks like the scan operator repr can be pretty long: https://github.com/Eventual-Inc/Daft/blob/b6d6669582c35552a4aa757dbad2263c49ce4266/src/daft-scan/src/anonymous.rs#L36

IMO each string in this returned vec should be pretty atomic/granular, and it should be up to the display mode (e.g. tree plan visualization vs. single-line summary) to condense it as desired.

Reply:
It does make it more readable imo! But the main reason to do this was to make it much closer to the Legacy Source repr to be able to write the repr based tests much easier. Both should be able about the same length